packages/getMyPrescriptions/tests/live-spine-client.test.ts

```diff
@@ -112,11 +129,10 @@ describe("live spine client", () => {
     "throw error when $scenarioDescription",
     async ({httpResponseCode, spineStatusCode, nhsdLoginUser, errorMessage}) => {
       mock.onGet("https://spine/mm/patientfacingprescriptions").reply(httpResponseCode, {statusCode: spineStatusCode})
-      const spineClient = new LiveSpineClient(logger)
       const headers: APIGatewayProxyEventHeaders = {
         "nhsd-nhslogin-user": nhsdLoginUser
       }
-      await expect(spineClient.getPrescriptions(headers)).rejects.toThrow(errorMessage)
+      expect(() => extractNHSNumber(headers["nhsd-nhslogin-user"])).toThrow(errorMessage)
```

Comment:
We don't need the mocking in these test cases any more. We also don't need the httpResponseCode and spineStatusCode.
I'd suggest a simpler version of the type spineFailureTestData, with only the fields that we need.

Reply:
Covered by removal of this test file, per Matt's comments.